alacritty.yml

```diff
@@ -218,6 +218,7 @@ mouse:
 
 selection:
   semantic_escape_chars: ",│`|:\"' ()[]{}<>"
+  semantic_detect_scheme: true
```

Comment:
I might call this `semantic_select_scheme` the way it's worded now makes me think Alacritty is going to do something special with the scheme.